wled00/data/settings_leds.htm

```diff
@@ -255,27 +305,32 @@
 <option value="43">PWM RGB</option>
 <option value="44">PWM RGBW</option>
 <option value="45">PWM RGBWC</option>
+<option value="10">DDP RGB (network)</option>
+<!--option value="11">E1.31 RGB (network)</option-->
+<!--option value="12">ArtNet RGB (network)</option-->
 </select>&nbsp;
-Color Order:
+<div id="co${i}" style="display:inline">Color Order:
 <select name="CO${i}">
 <option value="0">GRB</option>
 <option value="1">RGB</option>
 <option value="2">BRG</option>
 <option value="3">RBG</option>
 <option value="4">BGR</option>
 <option value="5">GBR</option>
-</select><br>
-<span id="p0d${i}">Pin:</span> <input type="number" class="xs" name="L0${i}" min="0" max="33" required onchange="UI()"/>
-<span id="p1d${i}">Clock:</span> <input type="number" class="xs" name="L1${i}" min="0" max="33" onchange="UI()"/>
-<span id="p2d${i}"></span><input type="number" class="xs" name="L2${i}" min="0" max="33" onchange="UI()"/>
-<span id="p3d${i}"></span><input type="number" class="xs" name="L3${i}" min="0" max="33" onchange="UI()"/>
-<span id="p4d${i}"></span><input type="number" class="xs" name="L4${i}" min="0" max="33" onchange="UI()"/>
+</select></div>
+<br>
+<span id="p0d${i}">GPIO:</span><input type="number" name="L0${i}" min="0" max="33" required class="xs" onchange="UI()"/>
+<span id="p1d${i}"></span><input type="number" name="L1${i}" min="0" max="33" class="xs" onchange="UI()"/>
+<span id="p2d${i}"></span><input type="number" name="L2${i}" min="0" max="33" class="xs" onchange="UI()"/>
+<span id="p3d${i}"></span><input type="number" name="L3${i}" min="0" max="33" class="xs" onchange="UI()"/>
+<span id="p4d${i}"></span><input type="number" name="L4${i}" min="0" max="33" class="xs" onchange="UI()"/>
 <br>
-<span id="psd${i}">Start:</span> <input type="number" name="LS${i}" id="ls${i}" min="0" max="8191" value="${lastEnd(i)}" required />&nbsp;
+<span id="psd${i}">Start:</span> <input type="number" name="LS${i}" id="ls${i}" class="l" min="0" max="8191" value="${lastEnd(i)}" disabled readonly required />&nbsp;
```

Comment:
Noooooo. 😢 
This is going to cause confusion (and it already is). Why not add reordering of buses if you want flexibility of positioning?
I think your _mirroring_ argument is not really valid since not all FX will render _mirrored_ even with the same start field.